### DISCUSSION TITLE
Cant upload avatar for super admin user.

### DISCUSSION THREAD

**Discussion by SattyKaur** [statamic/cms#5241] [Q&A]
I am having trouble saving an avatar for users with 'Super Admin' role on my live site. I am able to update avatars for non super admin users though and I have no problem on my local. 
I don't get any message apart from 'Something went wrong'. I have a couple of live Statamic sites and I am getting the same message on both sites. 

Our website uses the avatar for authors on our blogs, hence why I need them. Is anyone having the same problem or is able to help me. 

Thanks

**Comment by duncanmcclean** [comment]
Can you share your log file? It'll show the error message (`storage/logs`).

It might be worth deleting the log file, uploading an avatar, then copying in the logs in here. Just so we only see the error(s) related to the upload issue.

Thanks!
